# Route to confirm
geographic_msgs/GeoPath route
# Behaviour model
string model         # Base58
# Execution cost
uint32 cost          # int
# Deadline block number
uint32 deadline      # int